src/tools/rest-frontend/public/pages/main/website/home.html

```diff
@@ -136,6 +136,12 @@ <h4 class="list-group-item-heading pull-left">{{ post.title}}</h4>
                         <p class="list-group-item-text">{{ post.shortDesc}}</p>
                     </a>
                 </div>
+                <p><a href="https://github.com/ElektraInitiative/libelektra/releases/latest"><img src="https://img.shields.io/github/release/ElektraInitiative/libelektra.svg" alt="Release" /></a>
+                <a href="https://build.libelektra.org/jenkins/job/libelektra/job/master/lastBuild"><img src="https://img.shields.io/jenkins/t/https/build.libelektra.org/jenkins/job/libelektra/job/master.svg" alt="Jenkins Build Status" /></a>
+                <a href="https://travis-ci.org/ElektraInitiative/libelektra"><img src="https://travis-ci.org/ElektraInitiative/libelektra.svg?branch=master" alt="Travis Build Status" /></a>
+                <a href="https://cirrus-ci.com/github/ElektraInitiative/libelektra"><img src="https://api.cirrus-ci.com/github/ElektraInitiative/libelektra.svg" alt="Cirrus Build Status" /></a>
+                <a href="https://coveralls.io/github/ElektraInitiative/libelektra"><img src="https://img.shields.io/coveralls/github/ElektraInitiative/libelektra.svg" alt="Coverage Status" /></a>
+                <a href="https://lgtm.com/projects/g/ElektraInitiative/libelektra/alerts"><img src="https://img.shields.io/lgtm/alerts/g/ElektraInitiative/libelektra.svg?logo=lgtm&amp;logoWidth=18" alt="Total alerts" /></a></p>
```

Comment:
I'd rather format the HTML similar to the remaining code than having everything on one line as the lines are getting quite long. Especially the `<p>` tag should be on it's own line.

Reply:
Thank you for your quick review!